src/distributed_ls/Euclid/Vec_dh.c

```diff
@@ -29,7 +29,9 @@ void Vec_dhCreate(Vec_dh *v)
 void Vec_dhDestroy(Vec_dh v)
 {
   START_FUNC_DH
-  if (v->vals != NULL) FREE_DH(v->vals); CHECK_V_ERROR;
```

Comment:
Did you change the "semantic" of this line? Do we want to change?

Reply:
OK. I don't know the code. It seemed risky but I am fine with it if you know what it is doing. Thanks!